src/paths.cpp

```diff
@@ -66,6 +66,7 @@ wxString GetConfigFile()
 //     wxStandardPaths::GetResourcesDir
 //     wxStandardPaths::GetAppDir
 // Fortunately these should always return the exe directory.
+// TODO: Investigate using wxStandardPaths::DontIgnoreAppSubDir with wxWidgets 2.9.1+
 #if defined(__WXDEBUG__) && defined(__WXMSW__)
```

Comment:
Done.

FWIW, I put this TODO in as I wasn't sure whether you were okay going ahead with using wxWidgets 3.0+ across the board, so I wasn't sure we could rely on DontIgnoreAppSubDir being present. If we're comfortable with that, I suspect this whole code block could be simplified, as you could just turn off the logic that scans for debug/release in these paths. But I'd need to test on Windows to be sure, so I left it as is for now.